TNO.sln

```diff
@@ -1,92 +1,91 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio Version 16
-VisualStudioVersion = 16.0.30114.105
+# Visual Studio Version 17
+VisualStudioVersion = 17.5.33516.290
 MinimumVisualStudioVersion = 10.0.40219.1
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "api", "api", "{F627B24A-217D-4BF1-BC77-E1A92DBCD07F}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "TNO.API", "api\net\TNO.API.csproj", "{16EA028B-B4C8-416D-BE54-D73D75483668}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "TNO.API", "api\net\TNO.API.csproj", "{16EA028B-B4C8-416D-BE54-D73D75483668}"
```

Comment:
Someone's setup is corrupting these values.  I normally see this when Visual Studio is being used and one developer has one set of IDs and another developer has a set of different IDs.  Can someone sort this out?  @CelloLiu @kylgarmor

Reply:
I've reverted the changes here.  I will keep an eye on it going forward.